test-packages/integration-tests/test/auth-flows/auth-flow.spec.ts

```diff
@@ -49,9 +49,9 @@ describe('OAuth flows', () => {
     });
 
     const result = await BusinessPartner.requestBuilder()
-      .getAll()
-      .top(1)
-      .execute(destination!);
+        .getAll()
+        .top(1)
+        .execute(destination!);
```

Comment:
Why was this indentation changed? Has prettier changed some defaults or was it faulty beforehand?

Reply:
unintentional. Reverted